linkerd/app/inbound/src/direct.rs

```diff
@@ -90,6 +90,7 @@ impl<N> Inbound<N> {
             config,
             runtime: rt,
             stack: tcp,
+            tls_detect_metrics,
```

Comment:
I think we actually need to support tls detect metrics on the direct stack...

Reply:
mmkay, i can add that (plus a test?)